### PR TITLE
Removed risk score from UI elements

### PR DESCRIPTION
**Pull request by Omri-Levy**
## Summary by CodeRabbit

- **New Features**
  - Enhanced conditional styling in the Navbar component based on active states and filter conditions.
  - Improved data handling in the `useCaseInfoBlock` component by combining multiple data sources and refining the logic.

- **Bug Fixes**
  - Fixed layout inconsistencies in `CaseOverview` by updating grid column classes for better responsiveness.
  - Removed redundant imports and cleaned up rendering logic to prevent potential display issues.

- **Style**
  - Adjusted styling for the Risk Level header in the `OverallRiskLevel` component for a more consistent look.

- **Chores**
  - Updated subproject commit hash in `services/workflows-service/prisma/data-migrations` for better synchronization.

### CODE REVIEW

File: apps/backoffice-v2/src/common/components/molecules/BusinessReportSummary/BusinessReportSummary.tsx [hunk 1]
```diff
@@ -37,22 +36,6 @@ export const BusinessReportSummary: FunctionComponent<{
...
-                },
-                {
-                  'text-destructive': severity === Severity.CRITICAL,
-                },
-                'text-4xl font-bold',
-              )}
-              checkFalsy={false}
-            >
-              {riskScore}
-            </TextWithNAFallback>
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/common/components/molecules/BusinessReportSummary/BusinessReportSummary.tsx:55]: Post change.
image

File: apps/backoffice-v2/src/common/components/molecules/OverallRiskLevel/OverallRiskLevel.tsx [hunk 2]
```diff
@@ -29,22 +29,6 @@ export const OverallRiskLevel: FunctionComponent<{
...
-              },
-              {
-                'text-destructive': severity === Severity.CRITICAL,
-              },
-              'text-4xl font-bold',
-            )}
-            checkFalsy={false}
-          >
-            {riskScore}
-          </TextWithNAFallback>
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/common/components/molecules/OverallRiskLevel/OverallRiskLevel.tsx:47]: Post change.
image

File: apps/backoffice-v2/src/common/components/molecules/OverallRiskLevel/OverallRiskLevel.tsx [hunk 3]
```diff
@@ -65,7 +49,9 @@ export const OverallRiskLevel: FunctionComponent<{
           <TableHeader className={'[&_tr]:border-b-0'}>
             <TableRow className={'hover:bg-[unset]'}>
               <TableHead className={'h-0 ps-0 font-bold text-foreground'}>Risk Type</TableHead>
-              <TableHead className={'h-0 ps-0 font-bold text-foreground'}>Risk Level</TableHead>
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/common/components/molecules/OverallRiskLevel/OverallRiskLevel.tsx:68]: Added min width so the risk level header won't break.

File: apps/backoffice-v2/src/common/components/organisms/Header/Header.Navbar.tsx [hunk 4]
```diff
@@ -55,17 +55,18 @@ export const Navbar: FunctionComponent = () => {
...
-                            `gap-x-1 px-1.5 py-2 text-xs capitalize hover:bg-[#EBEEF9] hover:text-[#5E688E] active:bg-[#e0e4f6] [&:not([aria-current=page])]:text-[#8990AC]`,
-                            childNavItem.filterId
-                              ? {
-                                  'font-semibold text-[#20232E]':
-                                    childNavItem.filterId === filterId,
-                                  'text-[#8990AC] aria-[current=page]:font-normal':
-                                    childNavItem.filterId !== filterId,
-                                }
-                              : {},
-                          )}
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/common/components/organisms/Header/Header.Navbar.tsx:68]: Fixed some nav links not having an active state such as "profiles" and "merchant monitoring"

File: apps/backoffice-v2/src/lib/blocks/hooks/useCaseInfoBlock/useCaseInfoBlock.tsx [hunk 5]
```diff
@@ -15,7 +15,12 @@ export const useCaseInfoBlock = ({
   entityDataAdditionalInfo: TWorkflowById['context']['entity']['data']['additionalInfo'];
 }) => {
   return useMemo(() => {
-    if (Object.keys(entity?.data ?? {}).length === 0) {
+    const entityDetails = [
+      ...Object.entries(omitPropsFromObject(entity?.data, 'additionalInfo', 'address') ?? {}),
+      ...Object.entries(omitPropsFromObject(entityDataAdditionalInfo ?? {}, 'ubos')),
+    ];
+
+    if (Object.keys(entityDetails ?? {}).length === 0) {
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/lib/blocks/hooks/useCaseInfoBlock/useCaseInfoBlock.tsx:23]: Fixed empty entity info block state. Until now we checked length on unused keys.

File: apps/backoffice-v2/src/pages/Entity/components/Case/components/CaseOverview/CaseOverview.tsx [hunk 6]
```diff
@@ -52,7 +52,7 @@ export const CaseOverview = ({ processes }: { processes: string[] }) => {
   }
 
   return (
-    <div className="grid grid-cols-4 gap-4">
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/pages/Entity/components/Case/components/CaseOverview/CaseOverview.tsx:55]: Fixes elements overlapping on smaller screen sizes.

File: apps/backoffice-v2/src/pages/MerchantMonitoring/components/MerchantMonitoringTable/columns.tsx [hunk 7]
```diff
@@ -85,21 +84,6 @@ export const columns = [
...
-                ]]: riskScore || riskScore === 0,
-                'font-bold': riskScore || riskScore === 0,
-                'text-destructive': severity === Severity.CRITICAL || severity === Severity.HIGH,
-              },
-              'py-0.5',
-            )}
-            checkFalsy={false}
-          >
-            {riskScore}
-          </TextWithNAFallback>
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/pages/MerchantMonitoring/components/MerchantMonitoringTable/columns.tsx:102]: Post change.
image

**Review by MatanYadaev — APPROVED** [review]

**Pull request merged**